src/GovUk.Education.ExploreEducationStatistics.Content.Services/ThemeService.cs

```diff
@@ -25,7 +23,6 @@ public ThemeService(ContentDbContext contentDbContext)
             _contentDbContext = contentDbContext;
         }
 
-        [BlobCache(typeof(PublicationTreeCacheKey))]
         public async Task<IList<ThemeTree<PublicationTreeNode>>> GetPublicationTree(
```

Comment:
Should we add something to theme service to delete private cached data blocks when deleting a theme/topic during teardown (could create new ticket for that)?

Reply:
Done now thanks